Headers/AppKit/NSCollectionView.h

```diff
@@ -37,6 +40,22 @@
 
 @class NSCollectionViewItem;
 @class NSCollectionView;
+@class NSCollectionViewLayout;
+@class NSCollectionViewLayoutAttributes;
+@class NSCollectionViewTransitionLayout;
+@class NSPasteboard;
+@class NSNib;
+@class NSMapTable;
+@class NSMutableArray;
```

Comment:
You don't need any of these mutable classes, why do you define these?

Reply:
These are used in a number of places now in the code, so I'm not sure why you're saying they are unecessary.  I think this comment is outdated.